docs/content/docs/user-docs/cleanup.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: "Cleanup"
+title: "Uninstall ACK Controller"
 description: "Uninstalling an ACK controller"
 lead: ""
 draft: false
-menu: 
+menu:
   docs:
     parent: "installing"
 weight: 100
 toc: true
 ---
 
-## Uninstall ACK service controllers 
+## Uninstall ACK service controllers
```

Comment:
Don't need this title. On the page, the title is directly above it saying the same thing.